docs/getting-started/github-apps/github-apps.rst

```diff
@@ -70,6 +70,9 @@ For a workflow, the ``/.dockstore.yml`` has the following general structure
         subclass: <CWL | WDL | NFL | GXFORMAT2>
```

Comment:
I thought Noah had fixed this, but maybe there's another instance of it... It's not part of your PR, but since you're in the file, could you please change `GXFORMAT2` to `GALAXY`.